lib/active_storage/service.rb

```diff
@@ -1,4 +1,11 @@
 # Abstract class serving as an interface for concrete services.
+#
+# The available services are:
+#
+#  * +Disk+, to manage attachments saved directly on the hard drive.
+#  * +GCS+, to manage attachments through Google Cloud Storage.
+#  * +S3+, to manage attachments through Amazon S3.
+#  * +Mirror+, to be able to use several services to manage attachments.
```

Comment:
Should explain how to do configuration through Rails (since that too is hidden in the railtie) and directly.